api/src/pcapi/alembic/versions/20220909T094309_992529bd24e5_drop_recredit_constraint.py

```diff
@@ -0,0 +1,20 @@
+"""drop_recredit_constraint
+"""
+from alembic import op
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "992529bd24e5"
+down_revision = "6dbd91a3c97e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("ALTER TABLE recredit DROP CONSTRAINT IF EXISTS recredittype")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏On pourrait plutot ajouter `MANUAL` à la liste des recredit types acceptés ?

Reply:
on peut oui

Reply:
Je suis pas sûr, pour 2 raisons : 
- à la base je ne pense pas que cette contrainte recredittype était voulue. on ne voulait justement pas de contrainte d'enums en tout cas
- là il faudrait drop la contrainte puis la recréer, c'est pas génial je trouve ça fait un alter table donc possiblement du downtime